### DISCUSSION TITLE
For some reasons dotfiles are not visible

### DISCUSSION THREAD

**Discussion by PrathmeshVishwakarma** [nvim-tree/nvim-tree.lua#2795] [Q&A] [answered]
I think this is not a bug, so posting here. I am certain I am doing something wrong, 
but dotfiles are not showing in tree, I did the config 
here it is
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/170882272/c6e5fffd-c90c-45bd-a1fc-d397097a5ac6)
but the result are not showing, image below is my home directory
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/170882272/c52e3022-2f13-4be9-8ee7-80718b58d007)
Thanks for help

**Comment by alex-courtis** [comment, marked as answer]
Default `dotfiles` `false` will show them: it means disable the dotfiles filter. See `:help nvim-tree.filters.dotfiles`

Try the dotfiles filter toggle `H`.

Refer to live help: `g?`.

**Comment by PrathmeshVishwakarma** [comment]
oh, xD. thanks for the help. Cheers
